nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/ConnectionHandler.java

```diff
@@ -88,6 +88,7 @@ class ConnectionHandler implements Runnable {
 
     @Override
     public final void run() {
+        Thread.currentThread().setName("[" + socket.socketId() + " " + socket.childSocketId() + "]");
```

Comment:
Does the socketId already have a human friendly prefix like "nima-server-" ? If not, maybe  add that here.